### PR TITLE
HX tech team

### PR DESCRIPTION
**Pull request by jodiedoubleday**
## What does this PR do?
- Change the meta description for each blog post to mention a snippet of information about the post itself
- Add Carolines MD5 Gravatar
- Removed "by Holiday Extras" and "The Tech Team at Holiday Extras" from the header on all pages.

## How to test
- Checkout branch
- Run `jekyll serve --safe` this should boot up the blog locally
- Visit your IP address (localhost) on your phone (:4000/)
- Check the Meta Description and a few posts and default pages to ensure it makes sense
- Ensure all pictures of the team and now displayed

#### Quality checklist (for PR author to complete BEFORE code review):
- [x] I've updated documentation with any changes to procedures..
- [x] I am ready for this to be code reviewed, merged and tested.

#### IP or Developer Review:
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve checked for coding anti-patterns.
- [x] +1 from me!

#### Software Engineer, Software Architect or Developer review:
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve checked for coding anti-patterns.
- [ ] +1 from me!

### CODE REVIEW

**Comment by lukefrake** [comment]
Takeing reviewer 1

**Comment by lukehansell-hx** [comment]
I'm getting the first line of the posts as the description. Is that the expected behaviour?

**Comment by jodiedoubleday** [comment]
@lukehansell-hx  yep, trying to make the description be for each individual post rather than something generic

**Comment by dancork** [comment]
Taking review

**Comment by dancork** [comment]
Meta keywords "JS, Javascript, CSS, PHP, DevOps, Ops, Tech, Selenium, CSS, HTML, Holiday Extras, Development, Web, Website, Programming"

PHP? No Node/NodeJS, Agile.

File: _includes/head.html [hunk 1]
```diff
@@ -3,10 +3,10 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>Hungry Geek - {{ page.title }}</title>
-
+    {% assign author = site.data.members.[page.author] %}
+    <title>Hungry Geek - {{ page.title }} {% if author.name %}by {{ author.name }}{% endif %}</title>
```
Comment (dancork) [inline _includes/head.html]: Feel like we could so some more SEO with our titles. Even just using the main meta title. Maybe something for a future PR.

**Comment by dancork** [comment]
All looks good, :+1:  when points have been looked at

**Comment by jodiedoubleday** [comment]
Agree you point are valid and definitely for future PR's

**Pull request merged**